lib/FunctionIsolation/EnforceABI.cpp

```diff
@@ -159,6 +167,10 @@ class EnforceABIImpl {
   std::map<Function *, Function *> OldToNew;
   Function *FunctionDispatcher;
   Function *OpaquePC;
+  DenseMap<std::pair<GlobalVariable *, Function *>, AllocaInst *> CSVMap;
+  DenseMap<Function *, BasicBlock *> UnwindsMap;
```

Comment:
"Unwinds"?

Reply:
`UnwindEdgesMap` should make more sense.